### PR TITLE
Feature: Dynamic Cache Entry Expiry

### PR DESCRIPTION
**Pull request by TanmayDhobale**
This pull request enhances the existing Cache utility by introducing the capability to set dynamic expiry times fvor each cache entry. Previously the cache expiry time was statically set, limiting the utility's flexibility. With this update, users can specify the expiry time in seconds for each set operation, allowing for more nuanced cache control tailored to specific need.

### CODE REVIEW

File: src/db/Cache.ts [hunk 1]
```diff
@@ -1,50 +1,43 @@
...
   }
 
-  set(type: string, args: string[], value: any) {
-    this.inMemoryDb.set(`${type} ${JSON.stringify(args)}`, {
-      value,
-      expiry:
-        new Date().getTime() +
-        parseInt(process.env.CACHE_EXPIRE_S || '100', 10) * 1000,
-    });
+  set(type: string, args: string[], value: any, expiryInSeconds: number = 3600): void {
```
Comment (hkirat) [inline src/db/Cache.ts:16]: Should defualt to 
```
new Date().getTime() +
        parseInt(process.env.CACHE_EXPIRE_S || '100', 10) * 1000
```

**Review by hkirat — CHANGES_REQUESTED** [review]
Why so much lint diff?
Was the file not linted correctly before or is it bonked right now?

**Comment by TanmayDhobale** [comment]
@hkirat  ummm significant lint difference could indicate a file was not previously linted according to the current project's linting rules, or the linting configuration has  been updated or corrected can u pls check now

**Comment by TanmayDhobale** [comment]
Let's goooooo

**Pull request merged**